cps/src/main/js/workflow-editor.js

```diff
@@ -48,7 +48,34 @@ jenkinsJSModules.import('ace-editor:ace-editor-122')
                 });
 
                 editor.on('blur', function() {
-                    textarea.trigger('change');
```

Comment:
:bug: AFAICT this is still suppressing `doCheckScript`, which you now again rely on for script approval warnings.